tests/console/validate_zfcp_multipath.pm

```diff
@@ -33,6 +33,12 @@ sub verify_plain_scsi_block_devices {
     assert_equals 4, $small_disks, "Expected 4 SCSI block devices sized 42.9GB at LUN 0x4001405000000000, found $small_disks";
 }
 
+sub verify_plain_scsi_block_devices_2_block {
```

Comment:
I think you can replace the old function directly and update the description for the function for your change.

Reply:
Done